### ISSUE TITLE
Module has no exported member 'MODAL_MODAL_DIRECTVES' (same for BS_VIEW_PROVIDERS)

### ISSUE THREAD

**Issue by quiringk** [valor-software/ngx-bootstrap#587]
Why do I get 

Module has no exported member 'MODAL_MODAL_DIRECTVES'

and 

Module has no exported member 'BS_VIEW_PROVIDERS'

when trying to follow the Modal example using "ng2-bootstrap": "^1.0.17",   ??

**Comment by jhiemer** [comment]
@quiringk could you provide a code sample. I am running here with 1.0.17 and my import works with this snippet:

``` javascript
import { PAGINATION_DIRECTIVES, DROPDOWN_DIRECTIVES,
                 MODAL_DIRECTVES, BS_VIEW_PROVIDERS } from 'ng2-bootstrap/ng2-bootstrap';
```

**Comment by Ovidiu-S** [comment]
The issue is with 1.0.22
Install 1.0.17 and you will be fine

**Comment by Dinistro** [comment]
@Ovidiu-S This looks more like a messed up config than an issue in ng2-bootstrap. At the date of creation, 1.0.17 was the newest version.

@quiringk Is it still not working in your application?

**Comment by quiringk** [comment]
@Dinistro I installed the latest version (1.0.24) and am working on trying it again. If I import MODAL_DIRECTIVES I get "has no exported member 'MODAL_DIRECTIVES'. If I remove the 2nd 'I' in DIRECTIVES and import MODAL_DIRECTVES then there is no error. Will update in a little if I get it to work this time

**Comment by valorkin** [comment]
@quiringk can you do a clean install? this misprint was fixed a while ago

**Comment by valorkin** [comment]
@Ovidiu-S always use latest version

**Comment by quiringk** [comment]
@valorkin I tried running "npm uninstall ng2-bootstrap" and other similar commands (adding -g, --save) which wasn't removing the directory from node-modules. Then I looked around and rmdir'd on node-modules and then did a full reinstall of all the node modules and it worked. Thanks. I'm relatively new to node and angular2, but why didn't the uninstall ng2-bootstrap remove the directory from node-modules? In the future when a major change happens, do I have to remove node-modules and do a full reinstall?
